doc/source/performance/threadpool.rst

```diff
@@ -154,31 +154,47 @@ This variable can be used to define the number of threads that can use the CPU a
 
 The number of milliseconds before a running thread is considered stalled. When this limit is reached thread pool will wake up or create another thread. This is being used to prevent a long-running query from monopolizing the pool.
 
+.. rubric:: Upgrading from a version before 8.0.14 to 8.0.14 or higher 
+
+Starting with the release `8.0.14`, |Percona Server| uses the upstream
```

Comment:
Sorry, one more thing here: the rest of the file seem to use one-line-per-sentence structuring, while this paragraph here is split into several lines, at different lengths. Which is the correct (I'm not sure about documentation conventions)?

Laurynas also asked about this in an earlier comment.

Reply:
fixed.